src/compas/robots/model/link.py

```diff
@@ -168,7 +168,14 @@ def data(self, data):
         self.inertia = Inertia.from_data(data['inertia']) if data['inertia'] else None
 
 
-class Visual(Data):
+class Item(object):
```

Comment:
Perhaps the name `Item` could be specified more, I don't know, `LinkItem`, or `LinkComponent`, but since it's an internal class, it's ok for now, and we can refactor later if considered necessary.